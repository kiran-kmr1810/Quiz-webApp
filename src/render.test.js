import React from "react";
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-17-updated';
import renderer from 'react-test-renderer';
import {
    Link,
} from '@chakra-ui/react'

Enzyme.configure({ adapter: new Adapter() });


it('renders correctly', () => {
        const tree = renderer
          .create(<Link page="http://localhost:3000/">Quizzy</Link>)
          .toJSON();
        expect(tree).toMatchSnapshot();
});


it('renders correctly', () => {
        const tree = renderer
          .create(<Link page="http://localhost:3000/shome">shome</Link>)
          .toJSON();
        expect(tree).toMatchSnapshot();
});

it('renders correctly', () => {
    const tree = renderer
      .create(<Link page="http://localhost:3000/fhome">fhome</Link>)
      .toJSON();
    expect(tree).toMatchSnapshot();
});


it('renders correctly', () => {
    const tree = renderer
      .create(<Link page="http://localhost:3000/createquiz">createquiz</Link>)
      .toJSON();
    expect(tree).toMatchSnapshot();

});

it('renders correctly', () => {
    const tree = renderer
      .create(<Link page="http://localhost:3000/quiz">quiz</Link>)
      .toJSON();
    expect(tree).toMatchSnapshot();
});
