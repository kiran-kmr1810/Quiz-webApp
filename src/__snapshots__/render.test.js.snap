// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  className="chakra-link css-1u8qly9"
  page="http://localhost:3000/"
>
  Quizzy
</a>
`;

exports[`renders correctly 2`] = `
<a
  className="chakra-link css-1u8qly9"
  page="http://localhost:3000/shome"
>
  shome
</a>
`;

exports[`renders correctly 3`] = `
<a
  className="chakra-link css-1u8qly9"
  page="http://localhost:3000/fhome"
>
  fhome
</a>
`;

exports[`renders correctly 4`] = `
<a
  className="chakra-link css-1u8qly9"
  page="http://localhost:3000/createquiz"
>
  createquiz
</a>
`;

exports[`renders correctly 5`] = `
<a
  className="chakra-link css-1u8qly9"
  page="http://localhost:3000/quiz"
>
  quiz
</a>
`;
